ledger/src/blockstore_processor.rs

```diff
@@ -1379,28 +1374,26 @@ fn load_frozen_forks(
     timing: &mut ExecuteTimings,
     accounts_background_request_sender: &AbsRequestSender,
 ) -> result::Result<(u64, usize), BlockstoreProcessorError> {
-    let recyclers = VerifyRecyclers::default();
-    let mut all_banks = HashMap::new();
-    let mut last_status_report = Instant::now();
-    let mut pending_slots = vec![];
-    // The total number of slots processed
-    let mut total_slots_elapsed = 0;
-    // The total number of newly identified root slots
-    let mut total_rooted_slots = 0;
-    // The number of slots processed between status report updates
-    let mut slots_elapsed = 0;
-    let mut txs = 0;
     let blockstore_max_root = blockstore.max_root();
     let mut root = bank_forks.read().unwrap().root();
     let max_root = std::cmp::max(root, blockstore_max_root);
-
     info!(
         "load_frozen_forks() latest root from blockstore: {}, max_root: {}",
         blockstore_max_root, max_root,
     );
 
+    // The total number of slots processed
+    let mut total_slots_processed = 0;
```

Comment:
Switched `slots_elapsed` to `slots_processed`; I think `process` is a more accurate description and consistent with terminology elsewhere (ie the file name)